networks/p2p/dial.go

```diff
@@ -495,20 +496,20 @@ func (t *dialTask) Do(srv Server) {
 	if err != nil {
 		logger.Debug("[Dial] Failed dialing", "task", t, "err", err)
 		// Try resolving the ID of static nodes if dialing failed.
-		if _, ok := err.(*dialError); ok && t.flags&staticDialedConn != 0 {
-			if t.resolve(srv, t.dest.NType) {
-				if t.dest.TCPs != nil && len(t.dest.TCPs) != 0 {
-					err = t.dialMulti(srv, t.dest)
-				} else {
-					err = t.dial(srv, t.dest)
-				}
-				if err != nil {
-					t.failedTry++
-				}
+		if _, ok := err.(*dialError); ok && t.flags&staticDialedConn != 0 && t.resolve(srv, t.dest.NType) {
+			if t.dest.TCPs != nil && len(t.dest.TCPs) != 0 {
```

Comment:
```suggestion
			if len(t.dest.TCPs) > 0 {
```

Reply:
It will be changed in https://github.com/klaytn/klaytn/pull/811